misaligned.py

```diff
@@ -1,13 +1,15 @@
-
+global last_pair_number;
 def print_color_map():
     major_colors = ["White", "Red", "Black", "Yellow", "Violet"]
     minor_colors = ["Blue", "Orange", "Green", "Brown", "Slate"]
     for i, major in enumerate(major_colors):
         for j, minor in enumerate(minor_colors):
             print(f'{i * 5 + j} | {major} | {minor}')
-    return len(major_colors) * len(minor_colors)
-
+            # returning last pairnumber to check if it's 25
+    return len(major_colors) * len(minor_colors),i * 5 + j
 
-result = print_color_map()
+result,last_pair_number = print_color_map()

```

Comment:
the number of pairs can be verified. But how do we verify the correctness of the content in color map?
Please come up with a way to verify that too.